File: sqlLed.php
<!DOCTYPE html>
<html>

<head>
    <title>
        GPIO LED
    </title>
</head>

<body style="text-align:center;">

    <h1 style="color:green;">
        GPIO LED CONTROL
    </h1>


    <?php
        // Use an if statement to trigger php functions
        if(array_key_exists('offButton', $_POST)) { 
            offButton(); 
        } 
        else if(array_key_exists('onButton', $_POST)) { 
            onButton(); 
        } 
        // Define our php functions 
        function offButton() { 
            $servername = "localhost"; // Use localhost because file is already on the server
			$username = "root";
			$password = "guzzo";
			$dbname = "gpioControl"; // Database name

    // Create connection
    $conn = new mysqli($servername, $username, $password, $dbname);
    
    // Check connection
    if ($conn->connect_error) {
        die("Connection failed: " . $conn->connect_error);
    }

    // Update our value toggle led
    $sql = "UPDATE led SET toggleOne=1 "; 
    
    // Holds result of the query
    $result = $conn->query($sql); 

    // Tell user if the sql query failed 
    if ($conn->query($sql) === TRUE) {
        echo "Record successfully updated";
    }
    else {
        echo "Error: " .$sql . "<br>" . $conn->error;
    }
    
    //Close connection to database
    $conn->close(); 
        } 
        function onButton() { 
            $servername = "localhost"; // Use localhost because file is already on the server
			$username = "root";
			$password = "guzzo";
			$dbname = "gpioControl"; // Database name

    // Create connection
    $conn = new mysqli($servername, $username, $password, $dbname);
    
    // Check connection
    if ($conn->connect_error) {
        die("Connection failed: " . $conn->connect_error);
    }
    // Update value to toggle led
    $sql = "UPDATE led SET toggleOne=2 "; 
    
    // Holds result of the query
    $result = $conn->query($sql); 
    
    // Tell user if the sql query failed 
    if ($conn->query($sql) === TRUE) {
        echo "Record successfully updated";
    }
    else {
        echo "Error: " .$sql . "<br>" . $conn->error;
    }
    
    // Close connection to database
    $conn->close(); 
        }  
    ?>
    <!-- use a form to automatically execute our php functions -->
    <form method="post">
        <input type="submit" value="Off" class="button" name="offButton" />

        <input type="submit" value="On" class="button" name="onButton" />
    </form>
    </head>

</html>

File: sqlLED.py
# Imports
import pymysql
import RPi.GPIO as GPIO
import time 

# Setup GPIO settings
GPIO.setmode(GPIO.BCM) 
GPIO.setwarnings(False) 
GPIO.setup(21,GPIO.OUT) 

# Store string to compare to associative array
offdata = "((1,),)"

# Check if loop is running
print("Loop starting...") 

# Create a loop that constantly checks the database 
while 0 == 0:
	
# Open database connection
    db = pymysql.connect(host="localhost", user="root", passwd="guzzo", db="gpioControl")

# Create a cursor object using cursor() method
    cur = db.cursor()

# Select desired value from the table
    sql = "SELECT toggleOne FROM led"

    # Execute sql statement
    cur.execute(sql)
    
    # Get data from the sql statement and store in rows
    rows = cur.fetchall() 
    
    # Show us sql statement
    print(rows)
    
    # Stringify sql statement to compare with our offdata string
    r = str(rows)
    
    # Check our data to turn off light 
    if r == offdata:
        print("LED off") 
        GPIO.output(21,GPIO.LOW) # Turn off LED
        time.sleep(1)  # Add a delay to avoid constantly reconnect/disccnnecting
    else:
        print("LED on") 
        GPIO.output(21,GPIO.HIGH) # Turn on LED
        time.sleep(1) 

# Disconnect from server
    cur.close()
    db.close()
